elements/pl-file-editor/pl-file-editor.mustache

```diff
@@ -42,6 +42,19 @@ $(function() {
                             <label for="modal-{{uuid}}-themes">Editor Theme</label>
                             <select class="custom-select" id="modal-{{uuid}}-themes">
                             </select>
+
+                            <div id="modal-split" style="padding-top: 1rem"></div>
+                            <label for="modal-{{uuid}}-font-adjust">Font Size</label>
```

Comment:
There's no element in this modal with this ID, so this isn't a useful label. (This will be solved if you use `<select>` instead)